.github/workflows/test-only.yml

```diff
@@ -88,14 +88,53 @@ jobs:
 
       - name: Test changedetection.io container starts+runs basically without error
         run: |
-          docker run -p 5556:5000 -d test-changedetectionio
+          docker run --name test-changedetectionio -p 5556:5000  -d test-changedetectionio
           sleep 3
           # Should return 0 (no error) when grep finds it
           curl -s http://localhost:5556 |grep -q checkbox-uuid
           
           # and IPv6
           curl -s -g -6 "http://[::1]:5556"|grep -q checkbox-uuid
+          docker kill test-changedetectionio
+
+      - name: Test changedetection.io SIGTERM and SIGINT signal shutdown
+        run: |
+          
+          echo SIGINT Shutdown request test
+          docker run --name sig-test -d test-changedetectionio
+          sleep 3
+          echo ">>> Sending SIGINT to sig-test container"
+          docker kill --signal=SIGINT sig-test
+          sleep 3
+          # invert the check (it should be not 0/not running)
+          docker ps
+          docker logs sig-test
+          docker inspect --format '{{json .State.Running}}' sig-test
+          if [ $? -eq 0 ]
+          then
+            exit 1
+          fi
```

Comment:
please, consider this carefully.
https://github.com/dgtlmoon/changedetection.io/pull/1850#issuecomment-1764704028

Reply:
https://github.com/dgtlmoon/changedetection.io/actions/runs/6497238087/job/17645884252#step:14:81